src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2680,11 +2680,14 @@ bool SessionImpl::addTorrent_impl(const std::variant<MagnetUri, TorrentInfo> &so
     if (m_loadingTorrents.contains(id) || (infoHash.isHybrid() && m_loadingTorrents.contains(altID)))
         return false;
 
-    if (Torrent *torrent = findTorrent(infoHash); torrent)
+    if (Torrent *torrent = findTorrent(infoHash))
     {
         // a duplicate torrent is being added
         if (torrent->isPrivate())
+        {
+            LogMsg(tr("Torrent '%1' is already in the transfer list. Trackers cannot be merged because it is a private torrent.").arg(torrent->name()));
```

Comment:
Would it be better to not mention the transfer list?
```suggestion
            LogMsg(tr("Found existing torrent. Trackers cannot be merged because it is a private torrent.").arg(torrent->name()));
```

Reply:
> Would it be better to not mention the transfer list?

Just have used existing string from "Add new torrent dialog"... But your suggestion is better, IMO.

Reply:
Changed.